src/containers/Subject.js

```diff
@@ -549,10 +551,50 @@ class Subject extends Component {
     const wtsQc = wts.filter((r) => r.key.endsWith('multiqc_report.html'));
     const rnasum = wts.filter((r) => r.key.endsWith('RNAseq_report.html'));
 
-    const tsoCtdna = results_gds.filter((r) => r.path.includes('tso_ctdna'));
-    const tsoCtdnaBams = tsoCtdna.filter((r) => r.path.endsWith('bam'));
-    const tsoCtdnaVcfs = tsoCtdna.filter(
-      (r) => r.path.endsWith('vcf') || r.path.endsWith('vcf.gz')
+    // gds ICA
+
+    const wgsBams = results_gds.filter(
```

Comment:
Ideally you wouldn't need these filter, if you already do the search keyword when querying the api?

Reply:
Search keyword would need to make multiple API calls. At the mo, Subject API return flat; so we could instead improve it to return nested struct i.e. already filtered. One for GraphQL!